config/software/ruby-windows.rb

```diff
@@ -17,7 +17,17 @@
 name "ruby-windows"
 default_version "2.0.0-p451"
 
-if windows_arch_i386?
+fips_enabled = (project.overrides[:fips] && project.overrides[:fips][:enabled]) || false
+
+if fips_enabled
+  # We only currently support 32-bit FIPS builds
+  if windows_arch_i386?
+    relative_path "ruby-#{version}-i386-mingw32"
+    source url: "https://s3-us-west-2.amazonaws.com/yakyakyak/ruby-#{version}-i386-mingw32.7z"
```

Comment:
Yeah I wouldn't put any of this in "official" places.  It's just a stop-gap build.  The other monster PR I'm working on is the one that'll build these assets in omnibus.  No need to move things now.